processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -180,7 +180,7 @@ public int getIndex(String value)
           // Return -2 for non-null values to match what the BitmapIndex implementation in BitmapIndexColumnPartSupplier
```

Comment:
Please make this comment a javadoc comment of the enclosing method and make all class and method refs javadoc refs.

Reply:
done.